### PR TITLE
Dropdown: Update readme to include links/comments about focus behavior

### PR DESCRIPTION
**Pull request by micahgodbolt**
Updating dropdown readme to mention/link to focus discussion. 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8358)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Dropdown/docs/DropdownOverview.md [hunk 1]
```diff
@@ -1 +1,3 @@
...
\ No newline at end of file
+A Dropdown is a list in which the selected item is always visible, and the others are visible on demand by clicking a drop-down button. They are used to simplify the design and make a choice within the UI. When closed, only the selected item is visible. When users click the drop-down button, all the options become visible. To change the value, users open the list and click another value or use the arrow keys (up and down) to select a new value.
+
+Note that the Dropdown, like HTML Select elements and other [menus](https://www.w3.org/TR/wai-aria-practices-1.1/#menu), will select the first option on keyboard focus. See [this issue about placeholders](https://github.com/OfficeDev/office-ui-fabric-react/issues/5473) for more background, history and alternatives.
```
Comment (cliffkoh) [inline packages/office-ui-fabric-react/src/components/Dropdown/docs/DropdownOverview.md]: I think this blurp has potential to be contentious. We were specifically following listbox guidelines. I'm not even personally sure what menu specifically has to do with the Dropdown. The described behavior in HTML `select` is ambiguous (not sure what you're referring to). Additionally, the behavior varies by browser and platform (Safari's `<select>` works differently from IE)

Comment (JasonGore) [reply packages/office-ui-fabric-react/src/components/Dropdown/docs/DropdownOverview.md]: I was a bit too hasty to approve this but I do agree with Cliff. We still need to reassess Dropdown's ARIA role(s) and make sure its mapping cleanly into and conform with an ARIA role based on its configured props and mode. We should also reassess in consolidation with ComboBox.

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/Dropdown/docs/DropdownOverview.md]: prefer creating a new PR vs reverting and submitting the PR again? Happy to continue to iterate on these docs. The topic has just come up so many times I feel that some docs are better than nothing.

Comment (JasonGore) [reply packages/office-ui-fabric-react/src/components/Dropdown/docs/DropdownOverview.md]: Come to think of it we already [have a mechanism on the live page](https://developer.microsoft.com/en-us/fabric#/components/dropdown) to show users issues like this... maybe we should just be increasing awareness of it whenever anybody asks about Dropdown.

image

**Pull request merged**